test-src/manageworkertype_test.js

```diff
@@ -17,15 +17,17 @@ describe('provisioner worker type api', () => {
   var workerTypeChanged = _.clone(workerTypeDefinition);
   workerTypeChanged.maxCapacity = 15;
 
-  it('should be able to create a worker (idempotent)', async () => {
+  // Test is disabled for automated testing because it interacts with aws
```

Comment:
Unfortunately I'm pretty sure of it. Ran it through nock with request recording turned on and it was talking to it.
iirc it's checking to see if the ami exists. It's partway into the endpoints with names like updateWorkerType and createWorkerType.

On Wed, Apr 20, 2016 at 20:19, John Ford <notifications@github.com> wrote:
In test-src/manageworkertype_test.js [https://github.com/taskcluster/aws-provisioner/pull/71#discussion_r60521498] :

> @@ -17,15 +17,17 @@ describe('provisioner worker type api', () => {
>    var workerTypeChanged = _.clone(workerTypeDefinition);
>    workerTypeChanged.maxCapacity = 15;
>  
> -  it('should be able to create a worker (idempotent)', async () => {
> +  // Test is disabled for automated testing because it interacts with aws


it interacts with aws? are you sure?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub [https://github.com/taskcluster/aws-provisioner/pull/71/files/9e9fbd769c8e2ccff8e4f3b2f6063e62a7dbf482..e5cb01afaef6555f51195cbcd03712ec6c4a63ea#r60521498]